controller/src/main/java/org/jboss/as/controller/extension/ExtensionRegistry.java

```diff
@@ -126,27 +128,11 @@
     private final RunningModeControl runningModeControl;
     private final ManagedAuditLogger auditLogger;
     private final JmxAuthorizer authorizer;
+    private final Supplier<SecurityIdentity> securityIdentitySupplier;
     private final ConcurrentHashMap<String, SubsystemInformation> subsystemsInfo = new ConcurrentHashMap<String, SubsystemInformation>();
     private volatile TransformerRegistry transformerRegistry = TransformerRegistry.Factory.create();
     private final RuntimeHostControllerInfoAccessor hostControllerInfoAccessor;
 
-
-    /**
-     * Constructor
-     *
-     * @param processType the type of the process
-     * @param runningModeControl the process' running mode
-     * @param auditLogger logger for auditing changes
-     * @param authorizer hook for exposing access control information to the JMX subsystem
-     *
-     * @deprecated Remove once there has been a core release and we can update wildfly
-     */
-    @Deprecated
-    public ExtensionRegistry(ProcessType processType, RunningModeControl runningModeControl, ManagedAuditLogger auditLogger, JmxAuthorizer authorizer) {
```

Comment:
I can add it back but I think I removed it due to the comment.

> Remove once there has been a core release and we can update wildfly

I think we were keeping it for staging the core release and the wildfly upgrade rather than backwards compatibility.